packages/run-protocol/test/vaultFactory/test-liquidator.js

```diff
@@ -141,13 +142,17 @@ const setupAmmAndElectorate = async (t, aethLiquidity, runLiquidity) => {
   const space = setupBootstrap(t, timer);
   const { consume, instance } = space;
   installGovernance(zoe, space.installation.produce);
+  // TODO
```

Comment:
TODO **what?**

Reply:
hehe. I remember that was for DRYing out the next two lines. I plan to use https://github.com/Agoric/agoric-sdk/blob/358c8b7bef2c9a659344d72fd52aa80f14a93b68/packages/run-protocol/test/supports.js#L163-L167

Reply:
I'll save that for another PR. WDYT of it @dckc ?

Reply:
it's a reasonable idiom... I'm not sure it carries its weight as an exported function, where a reader has to switch context to learn what it does.

Presuming `const { entries } = Object`, it's a one-liner:

```js
  entries(installations).forEach(([key, installation]) => space.installation.produce[key].resolve(installation));
```

I don't have much of a problem making the function available, but I think there's a significant risk that  folks will open-code it rather than remember that it's available as a helper function.

The helper function should take just the `space.installation` part, not the whole `space`. POLA.